databricks_cli/sdk/service.py

```diff
@@ -352,6 +352,25 @@ def list_available_zones(self):
         _data = {}
     
         return self.client.perform_query('GET', '/clusters/list-zones', data=_data)
+    
+    def get_events(self, cluster_id, start_time=None, end_time=None, order=None, event_types=None,
```

Comment:
We shouldn't include this since it's not part of our change to add the Secrets API. My guess is clusters team added a new API endpoint, and this was autogenerated.

Reply:
+1 we should probably remove this